receiver/otlpreceiver/config.md

```diff
@@ -0,0 +1,91 @@
+# "otlp" Receiver Reference
```

Comment:
Maybe since we're calling this document a reference, the generated filename should be `reference.md`?